vent/io/devices/valves.py

```diff
@@ -146,7 +150,10 @@ def setpoint(self, setpoint):
         Args:
             setpoint (float): A number between 0 and 1 representing how much to open the valve
         """
-        self.duty = self.response(setpoint)
+        if not 0 <= setpoint <= 100:

```

Comment:
may want to check if between 0 and 1 in case someone is trying to set as a proportion?

Reply:
Yeah I probably should have checked for consistency on this one

Reply:
it can theoretically be 0.5/100